src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/StartedNode.kt

```diff
@@ -68,9 +70,20 @@ class StartedNode(
             resultsTransport = this.resultsTransport,
             unpackedProduct = this.unpackedProduct,
             monitoringProcesses = this.monitoringProcesses,
-            ssh = this.ssh
+            ssh = this.ssh,
+            jdk = this.jdk
         )
     }
 
+    fun stopNode() {
+        ssh.newConnection()
+            .use { ssh ->
+                ssh.safeExecute(
+                    "${jdk.use()}; ./${unpackedProduct}/bin/stop-jira.sh",
```

Comment:
QA check: Will that succeed if called twice? i.e. when we store dataset twice.

Reply:
It's `safeExecute`, so it will work even if `stop-jira.sh` is not idempotent